src/core/reopt.jl

```diff
@@ -213,8 +213,12 @@ function build_reopt!(m::JuMP.AbstractModel, p::REoptInputs)
 				@constraint(m, [ts in p.time_steps], m[:dvGridToStorage][b, ts] == 0)
 				@constraint(m, [t in p.techs.elec, ts in p.time_steps_with_grid],
 						m[:dvProductionToStorage][b, t, ts] == 0)
+				@constraint(m, [ts in p.time_steps], m[Symbol("dvStorageToGrid")][ts] == 0)  # if there isn't a battery, then the battery can't export power to the grid
```

Comment:
do we need this? dvStorageToGrid should have b index so if there's no storage than the dv would have dimension of 0. Even without b index, isn't it already constrained to the storage size which would be 0 if no storage?